client/landing/stepper/declarative-flow/internals/state-manager/store.ts

```diff
@@ -0,0 +1,49 @@
+import { useQuery, useQueryClient } from '@tanstack/react-query';
+import { getFlowFromURL } from 'calypso/landing/stepper/utils/get-flow-from-url';
+import { getSessionId } from 'calypso/landing/stepper/utils/use-session-id';
+import { FlowStateManifest } from './stepper-state-manifest';
+
+const PREFIX = 'stepper-state-item';
+const VERSION = 'v1';
+
+const PERSISTENCE_CONFIG = {
```

Comment:
Can you add a comment here just to give some help understanding? Especially for the `networkMode` that may be a little more obscure.